src/Command/BackUpCommand.php

```diff
@@ -55,170 +31,150 @@ protected function execute(InputInterface $input, OutputInterface $output)
-            $result_size = sizeof($result->sites);
-            $temp_site_list = array_merge($temp_site_list, $result->sites);
-            $page++;
-        }
-
-        print "\nFound " . sizeof($temp_site_list) . " total sites on SaaS.\n";
-        $site_list = array();
-        foreach ($temp_site_list as $key => $value) {
-            $result = json_decode($client->request('GET', 'sites/' . $value->id)->getBody());
-            if ($day_of_week == $FULL_DAY_TO_RUN) {
-                $site_list[] = $result;
-            } else if (isset($result->is_primary) && $result->is_primary) {
-                $site_list[] = $result;
-            }
-        }
-        print "\nUsing " . sizeof($site_list) . " sites on SaaS.\n";
-        $client = new Client([
-            'base_uri' => 'https://cloudapi.acquia.com/v1/',
-            'auth' => [$input->getOption('cloud-username'), $input->getOption('cloud-key')],
-        ]);
-        $sites = json_decode($client->request('GET', 'sites.json')->getBody());
-        $paas_size = 0;
-        foreach ($sites as $site) {
-            if(substr($site, 0, strlen('prod:')) === 'prod:') {
-                $paas_size++;
-                $endpoint = implode('/', ['sites', $site, 'envs', 'prod.json']);
-                $server = json_decode($client->request('GET', $endpoint)->getBody());
-                $paas = new \stdClass();
-                $paas->id = 0;
-                $paas->stack_id = 0;
-                $paas->site = $server->default_domain;
-                $paas->domains = array($server->default_domain);
-                $paas->root = '/var/www/html/'.$server->unix_username.'/docroot';
-                $paas->remote_host = $server->ssh_host;
-                $paas->remote_user = $server->unix_username;
-
-                $the_domains = array();
-
-                $endpoint = implode('/', ['sites', $site, 'envs', 'prod', 'domains.json']);
-                $domains = json_decode($client->request('GET', $endpoint)->getBody());
-                foreach ($domains as $domain) {
-                    $the_domains[] = $domain->name;
-                }
-
-                //Let's also add a *.govcms.gov.au URL for PaaS sites
-                $govcms_domain = strtok($paas->site, '.');
-                $the_domains[] = $govcms_domain.'.govcms.gov.au';
-
-                $paas->collection_domains = $the_domains;
-                $ignored = false;
-                foreach ($ignore as $item) {
-                    if(in_array($item, $paas->domains) || in_array($item, $paas->collection_domains)) {
-                        $ignored = true;
-                    }
-                }
-                if(!$ignored) {
-                    $site_list[] = $paas;
-                }
-            }
-        }
-        print "Using " . $paas_size . " sites on PaaS.\n";
-
-        print "Using " . sizeof($site_list) . " sites.\n";
+        // @TODO: is this check for Sunday vs regular days still needed?
```

Comment:
@paulkilla 
I'm not sure about this check, it seems to me that it backups all sites on a Sunday but only ones with `is_primary` on other days.
Unfortunately I don't know what the `is_primary` is. 
Do you know if that is still needed?

Reply:
This check/process is no longer needed, as more backups will now be held on Lagoon.  We originally backed up all (primary&secondary) sites nightly, but as the backup footprint grew, we moved to only a weekly backup of the secondary/clone sites, and the primaries nightly.

Reply:
@tobybellwood 
great! thanks for explanation, removed from code